bin/react16_post_rollout_cleanup.dart

```diff
@@ -0,0 +1,15 @@
+// Copyright 2019 Workiva Inc.
```

Comment:
I believe this executable needs to be added to pubspec.yaml.

Reply:
Yep, 💯. Forgot about that. Thank you!